scripts/submit-pr.sh

```
@@ -41,6 +41,68 @@ if [[ -z "$MODE" ]]; then
+    gh api "repos/${REPO}/pulls/${pr_number}/comments" \
+        --jq '.[] | select(.user.login | startswith("coderabbitai")) | "- \(.path):\(.line // .original_line // "?") \(.body | split("\n")[0] | gsub("^_⚠️ Potential issue_ \\| _🟡 Minor_"; "[MINOR]") | gsub("^_⚠️ Potential issue_ \\| _🔴 Major_"; "[MAJOR]"))"' 2>/dev/null || echo "  (none)"
+
+    echo ""
+
+    # Get review body and extract nitpicks
+    local review_body=$(gh api "repos/${REPO}/pulls/${pr_number}/reviews" \
+        --jq '[.[] | select(.user.login | startswith("coderabbitai"))] | last | .body // empty' 2>/dev/null)
+
+    if [[ -n "$review_body" ]]; then
+        # Extract actionable count (macOS-compatible)
+        local actionable=$(echo "$review_body" | sed -n 's/.*Actionable comments posted: \([0-9]*\).*/\1/p' | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)